client/js/views/templates/Home.js

```diff
@@ -1 +1,38 @@
-module.exports = function( { model } ) { return `<div>Home page stuff goes here</div>` }
+module.exports = function({ model }) { 
+	return `<div>
+	<img src="https://storage.googleapis.com/five-gallon/FutureDaysFarmLogo.svg"/> 
+	<div>Welcome Headline</div>
+	<div>allegan county, michigan</div>
+	<div>Praesent laoreet ornare ligula, ac accumsan turpis sagittis at.  Integer auctor egestas eleifend. Etiam luctus 
+		mattis justo, vitae fermentum libero euismod lacinia. Proin at consequat risus.  Praesent sollicitudin 
+		vestibulum felis, ut sodales enim.</div>
+	<div>
+		<div>
+			<img class='pic' src='https://storage.googleapis.com/five-gallon/Jam.jpg' alt='Jam'> 
+			<div class='text'>Nov. 12<br>Wild Berry Jam</div>
+		</div>
+		<div>
+			<img class='pic' src="https://storage.googleapis.com/five-gallon/Beets.jpg" alt="Beets"> 
+			<div class='text'>Oct. 4<br>We Got The Beets</div>
+		</div>
+		<div>
+			<img src="https://storage.googleapis.com/five-gallon/Squash.jpg" alt="Squash"> 
+			<div class='text'>Oct. 13<br>It's Time For Squash</div>
+		</div>
+	</div>
+	<div>
+		<div>
+			<img class='pic' src='https://storage.googleapis.com/five-gallon/Vines.jpg' alt='Jam'> 
+			<div class='text'>August. 19<br>Chicken In The Vine</div>
+		</div>
+		<div>
+			<img class='pic' src="https://storage.googleapis.com/five-gallon/Carrots.jpg" alt="Beets"> 
+			<div class='text'>July. 9<br>Cute Misfit Carrots</div>
+		</div>
+		<div>
+			<img src="https://storage.googleapis.com/five-gallon/Kale.jpg" alt="Squash"> 
```

Comment:
For images, we have a special function to load them in the background via ajax.  

Use:

```
<img data-src="${this.ImageSrc( value )}" />
```

Where value is the filename.